### PR TITLE
move kssh config from kbfs to kv store

### PR DESCRIPTION
**Pull request by mmou**
highlights of changes, other than replacing kbfs kssh config code with kvstore code:

- moved shared kbfs.go file into keybaseca.
- updated tests: change `keybase fs` run_commands to instead use `keybase kvstore`. also changed a couple of asserts checking for error strings relating to "config files". otherwise, lots of linting.
- updated documentation to not refer to "config files" in KBFS. on the files i touched, i also reformatted long lines (vim gq)
- updated circleci: cleaned up go linting, and added python linting

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
       - run: go test ./...
-  lint:
-    docker:
-      - image: circleci/golang:1.11
+  integration:
+    machine:
+      image: circleci/classic:latest
     steps:
       - checkout
-      - run: "! go fmt ./... 2>&1 | read"
```
Comment (mmou) [inline .circleci/config.yml:20]: golangci-lint covers this

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
-      - run: "! go vet ./... 2>&1 | read"
```
Comment (mmou) [inline .circleci/config.yml:21]: for all the other lines i removed: golangci-lint covers this, and the existing line (with the `! ....... | read`) wasn't doing anything

File: .isort.cfg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[settings]
```
Comment (mmou) [inline .isort.cfg:1]: needed to not conflict with black

File: src/cmd/kssh/kssh.go [hunk 4]
```diff
@@ -20,12 +19,12 @@ import (
 
 func main() {
 	kssh.InitLogging()
-	team, remainingArgs, action, err := handleArgs(os.Args[1:])
+	botName, remainingArgs, action, err := handleArgs(os.Args[1:])
```
Comment (mmou) [inline src/cmd/kssh/kssh.go:22]: was named wrong

File: src/cmd/kssh/kssh.go [hunk 5]
```diff
@@ -35,12 +34,18 @@ func main() {
 		doAction(action, keyPath, remainingArgs)
 		os.Exit(0)
 	}
-	config, err := getConfig(team)
+
+	bot, err := kssh.NewBot()
```
Comment (mmou) [inline src/cmd/kssh/kssh.go:38]: how this works:
first need to set up go chat bot
then in `Configure()`, need to use the bot api to get the kssh config from the KV store, to return a `ConfiguredBot`

File: src/keybaseca/bot/bot.go [hunk 6]
```diff
@@ -18,42 +22,47 @@ import (
...
+	return Bot{conf: conf, api: api}, nil
 }
 
-// Start the keybaseca bot in an infinite loop. Does not return unless it encounters an unrecoverable error.
-func StartBot(conf config.Config) error {
-	kbc, err := GetKBChat(conf)
+// Start the SSH CA bot in an infinite loop. Does not return unless it
+// encounters an unrecoverable error.
+func (b *Bot) Start() error {
+	err := b.writeClientConfig()
```
Comment (mmou) [inline src/keybaseca/bot/bot.go:43]: i moved the writing and deleting (and capturing SIGTERM) of the client config from src/cmd/keybaseca/keybaseca.go to here (in bot.go)

File: src/keybaseca/bot/bot.go [hunk 7]
```diff
@@ -131,25 +142,141 @@ func StartBot(conf config.Config) error {
...
+	return nil
+}
+
+// Attempts to delete the kssh configs for the specified teams.
+func (b *Bot) deleteClientConfig(teams []string) error {
+	var found []string
+	for _, team := range teams {
+		_, err := b.api.DeleteEntry(&team, shared.SSHCANamespace, shared.SSHCAConfigKey)
+		if err != nil {
+			if err.(kbchat.Error).Code == kbchat.DeleteNonExistentErrorCode {
```
Comment (mmou) [inline src/keybaseca/bot/bot.go:192]: if it failed to delete the config because the entry was not found, ignore the error

File: src/kssh/config.go [hunk 8]
```diff
@@ -6,114 +6,94 @@ import (
...
+		}
+		return &conf, nil
 	}
-	err = json.Unmarshal(bytes, &cf)
+	// then entry doesn't exist; no config found
+	return nil, nil
+}
+
+func (b *Bot) getAllTeams() (teams []string, err error) {
+	// TODO: dedup with same method in keybaseca/bot
```
Comment (mmou) [inline src/kssh/config.go:71]: not my highest priority...

File: src/kssh/config.go [hunk 9]
```diff
@@ -242,16 +198,51 @@ func GetDefaultBotAndTeam() (string, string, error) {
 	return lcf.DefaultBotName, lcf.DefaultBotTeam, nil
 }
 
-// Get the teamname associated with the given botname
-func GetTeamFromBot(botname string) (string, error) {
-	configs, _, err := LoadConfigs()
+// LoadConfigForBot gets the Config associated with the given botName. Will
+// need to find and load all configs from KV store.
+func (b *Bot) LoadConfigForBot(botName string) (Config, error) {
```
Comment (mmou) [inline src/kssh/config.go:203]: made more sense to return the whole config (instead of just the team name) in order to avoid another config load

File: src/kssh/configured_bot.go [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+	"github.com/keybase/go-keybase-chat-bot/kbchat"
+)
+
+type ConfiguredBot struct {
+	conf Config
+	api  *kbchat.API
+}
+
+// Get a signed SSH key from interacting with the CA chatbot
+func (cb *ConfiguredBot) GetSignedKey(request shared.SignatureRequest) (shared.SignatureResponse, error) {
```
Comment (mmou) [inline src/kssh/configured_bot.go:19]: only a configured bot can GetSignedKey

File: tests/bot-entrypoint.py [hunk 11]
```diff
@@ -1,9 +1,10 @@
 #!/bin/python3
 
 """
-This file is the main process running inside of the ca-bot container for the integration tests. This allows the kssh
-container to specify that the tests should run with a specific set of environment variables. This allows us to easily
-run integration tests for different keybaseca config options.
+This file is the main process running inside of the ca-bot container for the
```
Comment (mmou) [inline tests/bot-entrypoint.py:4]: linting...

File: tests/tests/conftest.py [hunk 12]
```diff
@@ -9,11 +9,43 @@ def run_around_tests():
...
+    # Automatically run after all tests in order to ensure that no kssh-client
+    # configs stick around
     tc = TestConfig.getDefaultTestConfig()
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh.staging/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh.prod/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh.root_everywhere/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam_secondary}/kssh-client.config || true" )
\ No newline at end of file
+    run_command(
```
Comment (mmou) [inline tests/tests/conftest.py:17]: changed to hit kvstore instead of fs

File: tests/tests/lib.py [hunk 13]
```diff
@@ -45,80 +60,117 @@ def run_command_with_agent(cmd: str) -> bytes:
...
 @contextmanager
 def simulate_two_teams(tc: TestConfig):
-    # A context manager that simulates running the given function in an environment with two teams set up
-    run_command(f"keybase fs read /keybase/team/{tc.subteam}.ssh.staging/kssh-client.config | "
-                     f"sed 's/{tc.subteam}.ssh.staging/{tc.subteam_secondary}/g' | "
-                     f"sed 's/{tc.bot_username}/otherbotname/g' | "
-                     f"keybase fs write /keybase/team/{tc.subteam_secondary}/kssh-client.config")
+    # A context manager that simulates running the given function in an
+    # environment with two teams set up
+    run_command(
```
Comment (mmou) [inline tests/tests/lib.py:143]: changed to use kvstore instead of fs

File: tests/tests/test_env_4_user_not_in_configured_teams.py [hunk 14]
```diff
@@ -2,37 +2,79 @@
...
+        ):
+            client_config = json.dumps(
+                {
+                    "teamname": f"{test_config.subteam}.ssh",
+                    "channelname": "",
+                    "botname": test_config.bot_username,
+                }
+            )
+            run_command(
+                f'echo \'{{"method":"put", "params": {{'
```
Comment (mmou) [inline tests/tests/test_env_4_user_not_in_configured_teams.py:57]: changed to use kvstore instead of fs

File: .isort.cfg [hunk 15]
```diff
@@ -0,0 +1,7 @@
...
+combine_as_imports = true
+include_trailing_comma = true
+line_length = 88
+multi_line_output = 3
+known_third_party = "pyotp,yaml"
```
Comment (xgess) [inline .isort.cfg:6]: we aren't using these libraries, are we?

File: docs/contributing.md [hunk 16]
```diff
@@ -63,46 +64,61 @@ GLOBAL OPTIONS:
...
-keybaseca uses the ssh-keygen binary in order to complete all key signing operations. 
+keybaseca uses the ssh-keygen binary in order to complete all key signing
+operations. 
 
 #### KBFS
 
-In order to ensure that keybaseca can run inside of docker (which does not support FUSE filesystems without adding
-the CAP_SYS_ADMIN permission), all KBFS interactions are done via `keybase fs ...` commands. This makes it so that 
+In order to ensure that keybaseca can run inside of docker (which does not
+support FUSE filesystems without adding the CAP_SYS_ADMIN permission), all KBFS
```
Comment (xgess) [inline docs/contributing.md:114]: is this still necessary?

File: docs/troubleshooting.md [hunk 17]
```diff
@@ -14,11 +14,12 @@ FORCE_WRITE=true make generate
 
 ## kssh is slow, but it works
 
-When kssh starts, it has to search every team you are in for a `kssh-client.config` file which specifies the information
-that is needed in order to communicate with the CA chatbot. If you are only in a few teams, this is relatively fast 
-(1-2 seconds for <10 teams) but this can become much slower as the number of teams increases (6 seconds for 100 teams
-in my benchmarks). This complex start up procedure can be avoided by setting a default bot via 
-`kssh --set-default-bot cabotname` which should reduce kssh's startup time considerably. 
+When kssh starts, it has to search the KV store for every team you are in for
```
Comment (xgess) [inline docs/troubleshooting.md:17]: i doubt this is slow anymore even for a lot of teams. i suspect we could rip out the whole `--set-default-bot` construct and flow. it's easier to leave it for now. just a passing thought.

File: src/kssh/config.go [hunk 18]
```diff
@@ -6,113 +6,93 @@ import (
...
-// If a user of kssh is in in multiple teams that are running the CA bot they can configure a default bot to communicate
-// with. Note that we store the team in here (even though it wasn't specified by the user) so that we can avoid doing
-// a call to `LoadConfigs` if a default is set (since `LoadConfigs can be very slow if the user is in a large number of teams).
-// This is controlled via `kssh --set-default-bot foo`.
+// If a user of kssh is in in multiple teams that are running the CA bot they
+// can configure a default bot to communicate with. Note that we store the team
+// in here (even though it wasn't specified by the user) so that we can avoid
+// doing a call to `LoadConfigs` if a default is set (since `LoadConfigs can be
+// very slow if the user is in a large number of teams).  This is controlled
+// via `kssh --set-default-bot foo`.
```
Comment (xgess) [inline src/kssh/config.go:90]:
```suggestion
// doing a call to `LoadConfigs` if a default is set. This is controlled
// via `kssh --set-default-bot foo`.
```

File: docs/troubleshooting.md (hunk 17 above)
Comment (xgess) [reply docs/troubleshooting.md:17]: maybe i'm missing something, but with this no longer an issue, there's really not much value to having a client config file at all. all of that persistent state isn't nearly as valuable as the added complexity of having some configs sometimes in a file and getting them there and checking for them. the way this thing is used, most people are going to be hitting the same servers and the same bot every time, so if kssh only exposed `kssh --bot yourcabot <rest of command>` in the event you were in multiple teams with ca-bots, you just write your own alias once (e.g. `kssh-prod`) and never think about it again, and then kssh gets the added benefit of being basically stateless (excepting your non-kssh ssh configs). 

not a task for this PR. but the move from kbfs->kvstore seems like it offers some additional opportunities.

File: src/cmd/kssh/kssh.go [hunk 19]
```diff
@@ -35,12 +34,18 @@ func main() {
...
+	if err != nil {
+		fmt.Printf("%v\n", err)
+		os.Exit(1)
+	}
+	cbot, err := bot.Configure(botName)
```
Comment (xgess) [inline src/cmd/kssh/kssh.go:43]: i think a lot of this is more confusing than it needs to be, and this line feels like a good example. first, none of this needs to be up at the level of the stack. pulling client configs so the kssh runner does the right thing can all be hidden down in the call to `provisionNewKey`. that would mitigate another thing i don't love here: the multiple use of `bot`. if i had my druthers, `bot` would only refer to the ca bot, and we'd use some other name (`runner` or `chatter` or `manager` or ...) to refer to the thing that manages chatting over keybase on behalf of the kssh user. bot on this line literally means two completely different things. 

and i have mixed opinions about the spinning out of a new object for the `configured` version of one of these things. i like that it's clearer what's what, and it allows separation of all of the configuration related methods from the actual heavy lifting of ssh work, but is there ever a reason anything might ever want the `unconfigured` version of one of these? calls to `New...` before configuring are already pretty heavy because they're setting up pipes to keybase. why not just do all of the configuration work right there (which i've mentioned above, i think i'd argue we should totally get rid of anyway) and make a cleaner interface to the caller right here?

tl;dr
```golang
func main() {
    caBotName, remainingArgs, action, err := handleArgs(os.Args[1:])
    ...
    provisionNewKey(caBotName, keyPath)
    doAction(action, keyPath, remainingArgs)
}

func provisionNewKey(caBotName, keyPath) {
    applicant := kssh.NewKsshApplicant(caBotName)  // naming things is hard!
    // ^ this object coming out right here would either be "configured"
    ...
    applicant.GetSignedKey()
    // ^ or "configuration" could happen as part of this call. either way, it doesn't really matter up here. 
    ....
}
```

File: src/keybaseca/bot/bot.go (hunk 7 above)
Comment (xgess) [reply src/keybaseca/bot/bot.go:192]: will this panic if the type coercion fails? that wouldn't be good.

File: src/cmd/keybaseca/keybaseca.go [hunk 20]
```diff
@@ -206,48 +203,25 @@ func beforeAction(c *cli.Context) error {
...
-				if exists {
-					err = constants.GetDefaultKBFSOperationsStruct().KBFSDelete(filename)
-					if err != nil {
-						fmt.Printf("%v\n", err)
-					}
-					teamsFoundMutex.Lock()
-					teamsFound = append(teamsFound, team)
-					teamsFoundMutex.Unlock()
-				}
-				semaphore.Done()
```
Comment (xgess) [inline src/cmd/keybaseca/keybaseca.go:235]: it's so awesome that we can delete all of this +1000

Comment (xgess) [reply src/cmd/keybaseca/keybaseca.go:235]: and +1 for encapsulating it in a method

File: src/cmd/kssh/kssh.go [hunk 21]
```diff
@@ -141,19 +146,19 @@ const (
...
+// Returns botName, remaining arguments, action, error
 // If the argument requires exiting after processing, it will call os.Exit
 func handleArgs(args []string) (string, []string, Action, error) {
 	remaining, found, err := kssh.ParseArgs(args, cliArguments)
 	if err != nil {
 		return "", nil, 0, fmt.Errorf("Failed to parse provided arguments: %v", err)
 	}
 
-	team := ""
+	botName := ""
```
Comment (xgess) [inline src/cmd/kssh/kssh.go:157]: this was likely a result of a change made a while back when we realized that the kssh-er specifying the team was strictly worse than specifying the bot. couple of things must have been missed in the change-over. but yeah - that's why this mistake is here.

File: src/keybaseca/bot/bot.go (hunk 6 above)
Comment (xgess) [reply src/keybaseca/bot/bot.go:43]: it's also not really relevant to the caller - which is why i assumed you pushed it down here. but both reasons seem good.

File: src/keybaseca/bot/bot.go [hunk 22]
```diff
@@ -131,25 +142,143 @@ func StartBot(conf config.Config) error {
...
+			// If they didn't configure a chat team, messages should be sent to any
+			// channel. This is done by having each client config reference the team
+			// it is found in.
+			config.TeamName = team
+			config.ChannelName = ""
+		}
+
+		var bytes []byte
+		bytes, err := json.Marshal(config)
+		if err != nil {
```
Comment (xgess) [inline src/keybaseca/bot/bot.go:173]: inside this error is a pretty good place to drop a log for which team had an issue

File: src/keybaseca/bot/bot.go [hunk 23]
```diff
@@ -131,25 +142,143 @@ func StartBot(conf config.Config) error {
...
+			return err
+		}
+		_, err = b.api.PutEntry(&team, shared.SSHCANamespace, shared.SSHCAConfigKey, string(bytes))
+		if err != nil {
```
Comment (xgess) [inline src/keybaseca/bot/bot.go:177]: here too

File: src/kssh/bot.go [hunk 24]
```diff
@@ -1,118 +1,75 @@
...
+func NewBot() (bot Bot, err error) {
 	runOptions := kbchat.RunOptions{KeybaseLocation: GetKeybaseBinaryPath()}
-	kbc, err := kbchat.Start(runOptions)
+	api, err := kbchat.Start(runOptions)
 	if err != nil {
-		return empty, fmt.Errorf("error starting Keybase chat: %v", err)
-	}
-
-	// Validate that the bot user is different than the current user
-	if config.BotName == kbc.GetUsername() {
```
Comment (xgess) [inline src/kssh/bot.go:25]: did this check disappear?

Comment (xgess) [reply src/kssh/bot.go:25]: i see it. nevermind.

File: tests/configure_accounts.py [hunk 25]
```diff
@@ -1,31 +1,37 @@
...
+        [random.SystemRandom().choice(string.ascii_lowercase) for _ in range(len)]
+    )
+
 
 def start_service():
     """
-    Start a new keybase service process in a randomly generated home directory and return the home directory
-    :return: str: The home directory to be used with future invocations of keybase commands
+    Start a new keybase service process in a randomly generated home directory
+    and return the home directory :return: str: The home directory to be used
```
Comment (xgess) [inline tests/configure_accounts.py:26]:
```suggestion
    and return the home directory 
    :return: str: The home directory to be used
```

File: tests/configure_accounts.py [hunk 26]
```diff
@@ -71,11 +84,13 @@ def create_team_and_invite(self, team, username):
     def create_channel(self, team, channel):
         """
         Create a channel in the given team
-        :param team:        The name of the team that the channel will reside in
+        :param team:        The name of the team that the channel will be in
```
Comment (xgess) [inline tests/configure_accounts.py:87]: 👍

File: tests/tests/conftest.py [hunk 27]
```diff
@@ -9,11 +9,43 @@ def run_around_tests():
...
+        f'echo \'{{"method": "del", '
+        f'"params": {{"options": {{"team": '
+        f'"{tc.subteam}.ssh", "namespace": "__sshca", '
+        f'"entryKey": "kssh_config"}}}}}}\' | '
+        f"xargs -0 -I del keybase kvstore api -m del || true"
```
Comment (xgess) [inline tests/tests/conftest.py:22]: eww. :/ suggest pulling this up into a method that builds the full command string from the name of the team. or whatever generally feels more readable and less repeated across all of these.

File: tests/tests/lib.py [hunk 28]
```diff
@@ -45,80 +60,117 @@ def run_command_with_agent(cmd: str) -> bytes:
...
+        f'"entryKey": "kssh_config"}}}}}}\' | '
+        f"xargs -0 -I get keybase kvstore api -m get | "
+        f"jq '.result' | "
+        f"sed 's/{tc.subteam}.ssh.staging/{tc.subteam_secondary}/g' | "
+        f"sed 's/{tc.bot_username}/otherbotname/g' | "
+        f"jq '. | "
+        f'{{"method":"put", "params": {{"options": {{"team": '
+        f'.teamName, "namespace": .namespace, "entryKey": .entryKey,'
+        f'"entryValue":.entryValue}}}}}}\' | '
+        f"xargs -0 -I put keybase kvstore api -m put"
```
Comment (xgess) [inline tests/tests/lib.py:155]: this is quite a bit harder to read than what was here before. if we can't figure out a way to make the kvstore json commands look a lot nicer, how about just two `run_command` calls. fetch, update in python-land, put.

File: tests/tests/test_env_4_user_not_in_configured_teams.py [hunk 29]
```diff
@@ -2,37 +2,79 @@
...
+                f'"options": {{"team": "{test_config.subteam}.ssh", '
+                f'"namespace": "__sshca", "entryKey": "kssh_config",'
+                f'"entryValue": {json.dumps(client_config)} }}}}}}\' | '
```
Comment (xgess) [inline tests/tests/test_env_4_user_not_in_configured_teams.py:60]: this is all pretty code-smell-y. i'm not exactly sure what makes sense, but something that's here should probably be in another method.

File: .isort.cfg (hunk 15 above)
Comment (mmou) [reply .isort.cfg:6]: removed

File: docs/contributing.md (hunk 16 above)
Comment (mmou) [reply docs/contributing.md:114]: added this sentence at the beginning of the paragaph: `"keybaseca supports logging to a local or KBFS file. "`

File: docs/troubleshooting.md (hunk 17 above)
Comment (mmou) [reply docs/troubleshooting.md:17]: Removing the local kssh config is a good idea for a future improvement!

File: tests/tests/test_env_4_user_not_in_configured_teams.py (hunk 29 above)
Comment (mmou) [reply tests/tests/test_env_4_user_not_in_configured_teams.py:60]: Ideally we'd be using the python keybase api bot.... To reduce additional work for now, I'll make some helper methods for these run_command calls.

File: src/keybaseca/bot/bot.go (hunk 7 above)
Comment (mmou) [reply src/keybaseca/bot/bot.go:192]: good catch, thanks. fixed

File: src/cmd/kssh/kssh.go (hunk 19 above)
Comment (mmou) [reply src/cmd/kssh/kssh.go]: i removed the ConfiguredBot abstraction
i renamed the keybase bot package and struct to `ca` and `CertAuthority`, respectively.
i renamed the kssh bot struct to `Requester`.
i moved `New` methods that create a keybase bot API out of the "Action" methods (`serviceAction` and `mainAction`), and wrapped them into helper methods
hope that addresses everything here, lmk

File: integrationTest.sh [hunk 30]
```diff
@@ -12,7 +12,7 @@ fi
 set +u
 if [ -z "$CIRCLECI" ]; then
   cd docs/
-  make html 2>&1 > /dev/null
+  #make html 2>&1 > /dev/null
```
Comment (xgess) [inline integrationTest.sh:15]: ?

File: src/cmd/kssh/kssh.go [hunk 31]
```diff
@@ -246,12 +235,21 @@ func isValidCert(keyPath string) bool {
 	return time.Now().After(validAfter) && time.Now().Before(validBefore)
 }
 
-// Provision a new signed SSH key with the given config
-func provisionNewKey(cbot kssh.ConfiguredBot, keyPath string) error {
+// Provision a new signed SSH key :with the given config
```
Comment (xgess) [inline src/cmd/kssh/kssh.go:238]:
```suggestion
// Provision a new signed SSH key with the given config
```

File: tests/tests/conftest.py [hunk 32]
```diff
@@ -14,38 +14,8 @@ def pytest_sessionfinish(session, exitstatus):
...
-        f"xargs -0 -I del keybase kvstore api -m del || true"
-    )
-    run_command(
-        f'echo \'{{"method": "del", '
-        f'"params": {{"options": {{"team": '
-        f'"{tc.subteam_secondary}", "namespace": "__sshca", '
-        f'"entryKey": "kssh_config"}}}}}}\' | '
-        f"xargs -0 -I del keybase kvstore api -m del || true"
-    )
+    run_delete_kvstore_command(f"{tc.subteam}.ssh")
```
Comment (xgess) [inline tests/tests/conftest.py:17]: good compromise here. muuuuuuuch easier to see what's going on

File: integrationTest.sh (hunk 30 above)
Comment (mmou) [reply integrationTest.sh:15]: accidental commit ><, removing

File: src/cmd/kssh/kssh.go [hunk 33]
```diff
@@ -239,11 +239,7 @@ func isValidCert(keyPath string) bool {
 func provisionNewKey(botName string, keyPath string) error {
 	log.Debug("Generating a new SSH key...")
 
-	bot, err := kssh.NewBot()
-	if err != nil {
-		return err
-	}
-	cbot, err := bot.Configure(botName)
+	requester, err := kssh.NewRequester()
```
Comment (xgess) [inline src/cmd/kssh/kssh.go:242]: 👍

File: src/cmd/keybaseca/keybaseca.go [hunk 34]
```diff
@@ -7,7 +7,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/keybase/bot-sshca/src/keybaseca/ca"
+	"github.com/keybase/bot-sshca/src/keybaseca/bot"
```
Comment (xgess) [inline src/cmd/keybaseca/keybaseca.go:10]: wait what? didn't you just change _to_ `ca` _from_ `bot`?

Comment (mmou) [reply src/cmd/keybaseca/keybaseca.go:10]: sorry if this was confusing -- i changed to ca then decided to change it back to bot, after re-reading your comment (`"if i had my druthers, bot would only refer to the ca bot"`).
